framework/update/tests/appupdateservice_tests.cpp

```
@@ -136,13 +141,35 @@ class AppUpdateServiceTests : public ::testing::Test, public ::async::Asyncable
         }));
     }
 
+    //! [GIVEN] An available release is ready to be downloaded.
+    void givenAvailableRelease(const std::string& fileName = "MuseScore.dmg",
```

Clang (14.0.6)

[warning] method 'givenAvailableRelease' can be made static

(readability-convert-member-functions-to-static)